pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -591,6 +591,23 @@ def test_missing_key_raises_keyerror2(self):
         with pytest.raises(KeyError, match=r"\(0, 3\)"):
             ser.loc[0, 3]
 
+    def test_missing_key_combination(self):
+        # GH: 19556
+        mi = MultiIndex.from_arrays(
+            [
+                np.array(["a", "a", "b", "b"]),
+                np.array(["1", "2", "2", "3"]),
+                np.array(["c", "d", "c", "d"]),
+            ],
+            names=["one", "two", "three"],
+        )
+        df = DataFrame(np.random.rand(4, 3), index=mi)
+        msg = r"\('b', '1', slice\(None, None, None\)\)"
+        with pytest.raises(KeyError, match=msg):
```

Comment:
can you also add the case where the 3rd level is level out (should also raise)

Reply:
Done